components/SaveFormData/SaveFormData.jsx

```diff
@@ -1,25 +1,66 @@
 import { useState, useEffect } from 'react';
 import Link from 'next/link';
 
-import { getDataIncludes } from 'utils/saveData';
+import { getDataIncludes, deleteData } from 'utils/saveData';
+import DeleteIcon from './DeleteIcon';
 
 export const SavedForms = () => {
   const [savedForms, setSavedForms] = useState();
   useEffect(() => {
     setSavedForms(getDataIncludes('/form'));
   }, []);
+  const deleteForm = (path) => {
+    deleteData(path);
+    setSavedForms(getDataIncludes('/form'));
+  };
   if (!savedForms) {
-    return <div>You don't have any incomplete form, well done!</div>;
+    return (
+      <p className="govuk-fieldset__legend--m gov-weight-lighter">
+        You don't have any incomplete form, well done!
+      </p>
+    );
   }
+  const tableHeader = [
+    { id: 'form_type', text: 'Form type' },
+    { id: 'last_saved', text: 'Last saved' },
+    { id: 'action', text: 'Actions' },
+    { id: 'delete', text: '' },
+  ];
+  const formQty = Object.entries(savedForms).length;
   return (
-    <ul className="govuk-list">
-      {Object.entries(savedForms).map(([key, value]) => (
-        <li key={key}>
-          <Link key={key} href={`${key}${value.step}&continueForm=true`}>
-            <a className="govuk-link">{key.replace(/(form)|(\/)/g, '')}</a>
-          </Link>
-        </li>
-      ))}
-    </ul>
+    <>
+      <p className="govuk-fieldset__legend--s gov-weight-lighter">
+        {`Displaying ${formQty} unfinished ${formQty > 1 ? 'forms' : 'form'}`}{' '}
+      </p>
+      <table className="govuk-table">
+        <thead className="govuk-table__head">
+          <tr className="govuk-table__row">
+            {tableHeader.map(({ id, text }) => (
+              <th key={id} className="govuk-table__header">
```

Comment:
Not a big deal but here we can use directly the `text` as `key`, so the `tableHeader` can become a list of strings.